### ISSUE TITLE
Does gizmo support windows？Crash on win7

### ISSUE THREAD

**Issue by supermario1990** [Lallassu/gizmo#2]
d:\go_code\src\github.com\Lallassu\gizmo>gizmo.exe
?[44;1m[17:10:26.351][main.go:31][main.run][DEBUG]?[0;1m W: Rect(0, 0, 800, 600)
?[44;1m[17:10:29.257][sound.go:23][main.(*sound).create][DEBUG]?[0;1m Loading sound jump.wav
?[44;1m[17:10:29.257][sound.go:23][main.(*sound).create][DEBUG]?[0;1m Loading sound shot.mp3
?[44;1m[17:10:29.549][sound.go:23][main.(*sound).create][DEBUG]?[0;1m Loading sound shot2.mp3
panic: oto: NewContext can be called only once

goroutine 20 [running]:
github.com/hajimehoshi/oto.NewContext(0xbb80, 0x2, 0x2, 0x31fc, 0x0, 0x0, 0x0)
        D:/go_code/src/github.com/hajimehoshi/oto/context.go:60 +0x2a8
github.com/hajimehoshi/oto.NewPlayer(0xbb80, 0x2, 0x2, 0x31fc, 0x31fc, 0x31fc, 0xc000609ff0)
        D:/go_code/src/github.com/hajimehoshi/oto/context.go:89 +0x54
github.com/faiface/beep/speaker.Init(0xbb80, 0xc7f, 0x0, 0x0)
        D:/go_code/src/github.com/faiface/beep/speaker/speaker.go:42 +0x16f
main.(*sound).create(0x7c70e0)
        D:/go_code/src/github.com/Lallassu/gizmo/sound.go:32 +0x53c
main.setup()
        D:/go_code/src/github.com/Lallassu/gizmo/main.go:66 +0x136
main.run()
        D:/go_code/src/github.com/Lallassu/gizmo/main.go:43 +0x297
github.com/faiface/mainthread.Run.func1(0x672540, 0xc00003e0c0)
        D:/go_code/src/github.com/faiface/mainthread/mainthread.go:37 +0x2e
created by github.com/faiface/mainthread.Run
        D:/go_code/src/github.com/faiface/mainthread/mainthread.go:36 +0xb8

**Comment by Lallassu** [comment]
I haven't tested it on other than OSX currently. I will plan to test it out later on Windows/OSX/Linux. Seems to be some issue with the sound-loader.

**Comment by mewmew** [comment]
re #3, this issue also happens on Linux.

**Comment by faiface** [comment]
This should be fixed with the recent update of Beep. See [here](https://github.com/faiface/beep/commit/fc2da0b5e081c0c574efd339344ad9f1bf649cba).

**Comment by faiface** [comment]
Okay, so I tried it and I cannot get past the extremely cool menu. Whenever I press New Game, everything freezes for a moment and then the menu is reset and the camera flies from the bottom to its original location and everything is as before.

Should I file a new issue?

**Comment by Lallassu** [comment]
Hehe press esc and the menu hides :)

As stated, it's a work in progress and much is not working as expected yet.

**Comment by mewmew** [comment]
> Hehe press esc and the menu hides :)

Hehe, yeah. I figured it out eventually. At first, I also couldn't gain access to the game :p
